inventories/terra-env/group_vars/docker-ol2.yml

```diff
@@ -1,6 +1,6 @@
 ---
 #overlay2 disk
-filesystems:
+docker-ol2_filesystems__to_merge:
```

Comment:
is the double underscore intentional?

Reply:
I believe so - I copied  this after mark's work.  I think the double underbar is a separator for the merged var addon.  https://github.com/broadinstitute/dsp-ansible-configs/blob/master/roles/ansible-role-clone-repos/tasks/main.yml

Reply:
yes it is intentional, its's part of the plugin